packages/core/addon/components/navi-visualizations/table.js

```diff
@@ -312,6 +353,177 @@ export default Component.extend({
     return direction;
   },
 
+  /**
+   * Synchronizes the width of visible table headers with real (hidden) table header elements
+   *
+   * @method _syncHeadersWidth
+   * @private
+   * @returns {void}
+   */
+  _syncHeadersWidth() {
+    //in editing mode the real table headers are visible, no need to sync
+    if (get(this, 'isEditingMode')) {
+      return;
+    }
+
+    let widths = this.$('thead th')
```

Comment:
So, I think we should move off of jquery. Community is trying to move away from it.  An alternative would be to use `this.element` which is native dom, and you can do querySelector/querySelectorAll off of it.